aws-sdk-java-v2-master/archetypes/archetype-lambda/src/main/resources/archetype-resources/src/main/java/DependencyFactory.java

```diff
@@ -26,7 +26,7 @@ private DependencyFactory() {}
 #if ($region == 'null')
                        .region(Region.of(System.getenv(SdkSystemSetting.AWS_REGION.environmentVariable())))
 #else
-                       .region(Region.${regionEnum})
+                       .region(Region.${regionEnum}))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;) Statement&#34; to complete IfStatement